.github/workflows/build.yml

```diff
@@ -112,10 +106,10 @@ jobs:
           - os: macos-14
             name: macOS-Legacy
             macosx_deployment_target: 10.13
-            qt_ver: 5
+            qt_ver: 6
             qt_host: mac
-            qt_version: "5.15.2"
-            qt_modules: "qtnetworkauth"
+            qt_version: "6.4.3"
+            qt_modules: "qt5compat qtimageformats qtnetworkauth"
```

Comment:
Qt 6.8 requires macOS 12 or newer.

Reply:
You need to adjust the deployment target for our main macOS build too.

Reply:
this is out of the scope of this PR and should be addressed at https://github.com/PrismLauncher/PrismLauncher/pull/2895